index.js

```diff
@@ -65,7 +65,7 @@ app.get('/recipe/:id', (req, res) => {
         const index = parseInt(req.params.id);
         const recipe = obj.recipes;
         const data = {
-            num: index,
+            id: index,
```

Comment:
Should you implement sort functionality for recipes eventually, consider creating an id property for each recipe object. Presently, an 'id' is only assigned to the props object for the purposes of rendering, and also at the routing stage, where the 'id' is correlated to the position of the recipe in its array. This correlation will be broken once any sorting logic is implemented, so a persistent id property may be helpful.